.github/workflows/erlang-tests.yml

```
@@ -0,0 +1,148 @@
+  test:
+    name: OTP ${{ matrix.otp }} – Ubuntu
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        otp: [24, 25, 26]
+
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)